test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -274,7 +274,6 @@
     "manageInstitutionalWallets": "boolean",
     "disabledUpgradeAccountsByChain": "object"
   },
-  "QueuedRequestController": { "queuedRequestCount": 0 },
```

Comment:
I think we need a migration to remove this state?

Reply:
Let me do this in a following PR, it does not harm anyways.  I would prefer to move this bigger PR sooner.

Reply:
👍